### ISSUE TITLE
You wanted to install npm 3.2.0, but the installed version is1.4.9. issue

### ISSUE THREAD

**Issue by rudym** [felixrieseberg/npm-windows-upgrade#23]
```
You wanted to install npm 3.2.0, but the installed version is1.4.9.
Please consider reporting your trouble to http://aka.ms/npm-issues.
Here is the output from the upgrader script:
[ '\r\n',
  '\r\n',
  '    Directory: C:\\Users\\<username>\\AppData\\Local\\Temp\r\n',
  '\r\n',
  '\r\n',
  'Mode                LastWriteTime     Length Name
  \r\n',
  '----                -------------     ------ ----
  \r\n',
  'd----         27-Jul-15   5:47 PM            npm_upgrade
  \r\n',
  'npm@3.2.0 node_modules\\npm\n',
  'All done!\r\n',
  '\r\n',
  '\r\n' ] []
```

**Comment by felixrieseberg** [comment]
Hey @rudym, thanks for reporting! Would you mind telling me which version of Windows and Node you're running?

**Comment by felixrieseberg** [comment]
Actually, I have hopes that the just released version 0.3.2 will fix your issue. Do me a favour and run `npm install -g npm-windows-upgrade` and `npm-windows-upgrade` again.

**Comment by rudym** [comment]
Hi @felixrieseberg , sorry for the inconvenience. The trouble was caused by `chocolatey`, it stored its copy of `npm`.
